tests/test_metadata.py

```diff
@@ -114,6 +114,23 @@ def test_basic(tmp_path: Path) -> None:
     }
 
 
+def test_no_git(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.setenv("PATH", "/foo")  # make sure git is not in PATH
+    assert _m(tmp_path) == {
+        "name": "odoo14-addon-addon1",
+        "version": "14.0.1.0.0",
+        "summary": "Addon 1",
+        "requires_dist": ["odoo>=14.0a,<14.1dev"],
+        "requires_python": ">=3.6",
+        "classifier": [
+            "Programming Language :: Python",
+            "Framework :: Odoo",
+            "Framework :: Odoo :: 14.0",
+        ],
+        "metadata_version": "2.1",
+    }
```

Comment:
```suggestion
    assert _m(tmp_path)["version"] == "14.0.1.0.0"
```

So the test is more focused on the only thing that depends on git.